common/src/main/java/com/wynntils/features/utilities/TranscribeMessagesFeature.java

```diff
@@ -85,19 +90,27 @@ public void onNpcDialogue(NpcDialogEvent event) {
 
         if (!transcribeWynnic && !transcribeGavellian) return;
 
-        event.setCanceled(true);
+        if (!showTooltip.get()) {
+            event.setCanceled(true);
+        }
 
         List<Component> transcriptedComponents = event.getChatMessage().stream()
                 .map(styledText -> getStyledTextWithTranscription(
                         StyledText.fromComponent(styledText), transcribeWynnic, transcribeGavellian, true))
                 .map(s -> ((Component) s.getComponent()))
                 .toList();
 
-        Managers.TickScheduler.scheduleNextTick(() -> {
-            NpcDialogEvent transcriptedEvent =
-                    new WynnTranscriptedNpcDialogEvent(transcriptedComponents, event.getType(), event.isProtected());
-            WynntilsMod.postEvent(transcriptedEvent);
-        });
+        if (showTooltip.get()) {
+            for (Component transcriptedComponent : transcriptedComponents) {
```

Comment:
Not 100% sure on this, just sending `transcriptedComponents.get(0)` seemed to work fine, but since it is a list this seemed better.

Reply:
Yeah getChatMessage() is a List<Component>

Reply:
Apparently we can in theory send more than one Component as NPC message. I coded this myself, but I don't remember if this actually happened, or if it was just a safety measure. But yes, better to keep it this way. It does not hurt.